keepassxc-browser/options/options.js

```diff
@@ -156,6 +156,16 @@ options.showKeePassXCVersions = function(response) {
     $('#tab-general-settings button.checkUpdateKeePassXC:first').attr('disabled', false);
 };
 
+options.getPartiallyHiddenKey = function(key) {
+    if (!key) {
+        return 'Error';
+    }
+
+    let hiddenKey = key.substr(0, 8);
+    hiddenKey += '*'.repeat(key.length - 8);
```

Comment:
Recommend just showing a fixed number of asterisks (10?)

Reply:
I agree. No need to count the length.

Reply:
I'd prefer five. Ten seems excessive.

Reply:
What matters is that user understands this is clearly a partially hidden key, not some black magic.

Reply:
I don't think that matters much. Users also understand when their credit card numbers are partially hidden and you will rarely see more than four asterisks for that.